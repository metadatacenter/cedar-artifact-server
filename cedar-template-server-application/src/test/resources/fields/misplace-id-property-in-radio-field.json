{
  "@type": "https://schema.metadatacenter.org/core/TemplateField",
  "@context": {
    "xsd": "http://www.w3.org/2001/XMLSchema#",
    "pav": "http://purl.org/pav/",
    "oslc": "http://open-services.net/ns/core#",
    "schema": "http://schema.org/",
    "pav:createdOn": {
      "@type": "xsd:dateTime"
    },
    "pav:createdBy": {
      "@type": "@id"
    },
    "pav:lastUpdatedOn": {
      "@type": "xsd:dateTime"
    },
    "oslc:modifiedBy": {
      "@type": "@id"
    }
  },
  "type": "object",
  "title": "Pick an option field schema",
  "description": "Pick an option field schema autogenerated by the CEDAR Template Editor ",
  "_ui": {
    "title": "Pick an option",
    "description": "",
    "inputType": "radio"
  },
  "_valueConstraints": {
    "requiredValue": false,
    "literals": [
      {
        "label": "Option 1"
      },
      {
        "label": "Option 2"
      },
      {
        "label": "Option 3"
      }
    ],
    "multipleChoice": false
  },
  "properties": {
    "@type": {
      "oneOf": [
        {
          "type": "string",
          "format": "uri"
        },
        {
          "type": "array",
          "minItems": 1,
          "items": {
            "type": "string",
            "format": "uri"
          },
          "uniqueItems": true
        }
      ]
    },
    "@id": {
      "type": [
        "string",
        "null"
      ],
      "format": "uri"
    },
    "_valueLabel": {
      "type": [
        "string",
        "null"
      ]
    },
    "pav:createdOn": {
      "type": [
        "string",
        "null"
      ],
      "format": "date-time"
    },
    "pav:createdBy": {
      "type": [
        "string",
        "null"
      ],
      "format": "uri"
    },
    "pav:lastUpdatedOn": {
      "type": [
        "string",
        "null"
      ],
      "format": "date-time"
    },
    "oslc:modifiedBy": {
      "type": [
        "string",
        "null"
      ],
      "format": "uri"
    }
  },
  "required": [
    "@value"
  ],
  "pav:createdOn": "2017-04-06T10:39:02-0700",
  "pav:createdBy": "https://metadatacenter.org/users/67a894dc-e9df-42d0-bb6e-5dbb4aa9dd79",
  "pav:lastUpdatedOn": "2017-04-06T10:39:02-0700",
  "oslc:modifiedBy": "https://metadatacenter.org/users/67a894dc-e9df-42d0-bb6e-5dbb4aa9dd79",
  "additionalProperties": false,
  "@id": "https://repo.metadatacenter.orgx/template-fields/6a35434a-d0f3-4670-9a61-2ac2f3f8246a",
  "$schema": "http://json-schema.org/draft-04/schema#"
}