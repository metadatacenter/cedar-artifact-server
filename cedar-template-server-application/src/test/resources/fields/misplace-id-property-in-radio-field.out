{"validates":"false","warnings":[],"errors":["A template field without value constraints must have a '@value' and not an '@id' field at path /properties/"]}